editor/src/communication/dispatcher.rs

```diff
@@ -93,7 +93,7 @@ impl Dispatcher {
 				}
 				Frontend(message) => {
 					// Image and font loading should be immediately handled
-					if let FrontendMessage::UpdateImageData { .. } | FrontendMessage::TriggerFontLoad { .. } = message {
+					if let FrontendMessage::UpdateImageData { .. } | FrontendMessage::TriggerFontLoad { .. } | FrontendMessage::TriggerRefreshBoundsOfViewports = message {
 						self.responses.push(message);
 						return;
 					}
```

Comment:
If we put the `push` line before the `if` statement, with only the `return;` inside, does that keep the logic the same? Assuming so, that would remove a redundant line and help clarify the logic. (If that wouldn't be logically equivalent, then I'm very confused by how this works, what's the difference between the two `push` lines?)

Reply:
The current way keeps the borrow checker happy with only a very simple line duplicated.